engines/ic3ia.cpp

```diff
@@ -63,12 +63,14 @@ IC3Formula IC3IA::get_model_ic3formula() const
 {
   TermVec conjuncts;
   conjuncts.reserve(predset_.size());
+  Term val;
   for (const auto &p : predset_) {
-    if (solver_->get_value(p) == solver_true_) {
+    if ((val = solver_->get_value(p)) == solver_true_) {
```

Comment:
just to be sure, what are you checking here?

Reply:
Yeah good question. This shouldn’t be an issue but it’s just a sanity check that the value is actually a value, like `true` or `false`. When I tried setting bool model generation only, it wouldn’t give a value for all terms, it would just return the same term.